archiver_test.go

```diff
@@ -140,6 +141,42 @@ func (s *ArchiverSuite) TestFixtures() {
 	}
 }
 
+func (s *ArchiverSuite) TestBad() {
```

Comment:
Ouch! That was not supposed to be in the PR. It's just a local test I use to test repositories that fail.